interface/python/mutationpp/__init__.py

```diff
@@ -1 +1,11 @@
-from ._mutationpp import *
+from ._mutationpp import *  # noqa: F403 # ty: ignore[unresolved-import]
+
+import os
+
+if "MPP_DATA_DIRECTORY" not in os.environ:
+    mpp_directory = os.path.dirname(__file__)
+    mpp_data_directory = os.path.join(mpp_directory, "data")
+    os.environ["MPP_DATA_DIRECTORY"] = mpp_data_directory
```

Comment:
Is there anyway now that we have it inside the wheel to pass this in a safer way instead of an env var?